src/js/components/search/visualizations/rank/RankVisualizationScopeButton.jsx

```diff
@@ -34,20 +34,14 @@ export default class RankVisualizationScopeButton extends React.Component {
             description += ' (currently selected)';
         }
 
-        let clickEvent = '';
-        if (this.props.label == 'Offices') {
-            clickEvent = null;
-        } else {
-            clickEvent = this.clickedButton;
-        }
-
         return (
             <button
                 className={`scope-button${activeClass}`}
                 value={this.props.value}
                 title={description}
                 aria-label={description}
-                onClick={clickEvent}>
+                onClick={this.clickedButton}
+                disabled={this.props.label === 'Offices' ? true : false}>
```

Comment:
Overcomplex ternary operations are [not allowed](http://eslint.org/docs/rules/no-unneeded-ternary) by our linter. You can switch this line to a simpler version:

```
disabled={this.props.label !== 'Offices'}>
```